app/Http/Controllers/Admin/ColumnMonsterCrudController.php

```diff
@@ -0,0 +1,115 @@
+     *
+     * @see  https://backpackforlaravel.com/docs/crud-operation-list-entries
+     *
+     * @return void
+     */
+    public function setupListOperation()
+    {
+        $this->crud->disableResponsiveTable();
+
+        $timeSpaceColumns = static::getFieldsArrayForTimeAndSpaceTab();
+        // Changed "time_range" column definition (changed "name" in comma separeated & convert "name" key in column definition)
+        if ($timeSpaceColumns) {
+            foreach ($timeSpaceColumns as $columnKey => $timeSpaceColumn) {
+                if ($timeSpaceColumn['type'] == 'date_range') {
+                    $dateRangeFieldName = $timeSpaceColumn['name'];
+                    $dateRangeColumnName = implode(',', $dateRangeFieldName);
+                    unset($timeSpaceColumns[$columnKey]);
+
+                    // Creating new variable array to over-ride date_range column as that is "unset" above
+                    $timeSpaceColumnDateRange = ['name' => $dateRangeColumnName, 'label' => $timeSpaceColumn['label'], 'type' => $timeSpaceColumn['type']];
+                    $timeSpaceColumns[$columnKey] = $timeSpaceColumnDateRange;
+                }
+            }
+        }
+
+        $relationshipColumns = static::getFieldsArrayForRelationshipsTab();
+        // Removing "custom_html" column definition
+        if ($relationshipColumns) {
+            foreach ($relationshipColumns as $columnKey => $relationshipColumn) {
+                if (isset($relationshipColumn['type']) && ($relationshipColumn['type'] == 'custom_html')) {
+                    unset($relationshipColumns[$columnKey]);
+                }
+            }
+        }
+
+        $miscellaneousColumns = static::getFieldsArrayForMiscellaneousTab();
+        // Adding extra attributes in "range" column definition
+        if ($miscellaneousColumns) {
+            foreach ($miscellaneousColumns as $columnKey => $miscellaneousColumn) {
+                if ($miscellaneousColumn['type'] == 'range') {
+                    // Creating new variable array to over-ride date_range column as that is "unset" above
+                    $miscColumnRange = [
+                        'name'              => $miscellaneousColumn['name'],
+                        'label'             => $miscellaneousColumn['label'],
+                        'type'              => $miscellaneousColumn['type'],
+                        'progress_class'    => 'bg-success',
+                        'is_striped'        => '1',
+                        'attributes'        => $miscellaneousColumn['attributes'],
+                        'tab'               => $miscellaneousColumn['tab'],
+                        'wrapperAttributes' => $miscellaneousColumn['wrapperAttributes'],
+                    ];
+
+                    $miscellaneousColumns[$columnKey] = $miscColumnRange;
+                } elseif ($miscellaneousColumn['type'] == 'color' || $miscellaneousColumn['type'] == 'color_picker') {
```

Comment:
Why are we doing custom stuff for `color` and `color_picker`? They should work by default, with the syntax we used inside the `color` and `color_picker` fields.

Reply:
We added few options for showing hex code & other suggested changes.